src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,220 @@
+-   fewer than seven top-level pages
+-   a flat navigation hierarchy
+-   limited space - recommended for HMI devices
 
-### Content-Cover
+<ImageGrid images={[NavigationRailImg]} regularWidth />
 
-<ImageGrid images={[TemporaryImg]} />
+## Persistent
 
-<!-- @TODO switch side-navigation links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+A persistent drawer is an always-visible drawer that can be opened and closed (expanded or collapsed). In its opened state, it looks and behaves like a permanent drawer. It the closed state, it looks and behaves like a navigation rail. The drawer is typically opened and closed by clicking an icon in the header. It can also be opened momentarily on desktop applications by hovering with the mouse. Use this variation if you have:
 
-<DemoCard
-    repository={'side-navigation'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-drawer--with-full-config'}
-    react
-/>
+-   fewer than seven top-level pages
+-   primary pages with labels that require complex icons or may be too long to wrap effectivly
+-   limited space but require a nested navigation hierarchy
+
+<React.Fragment>
+    <ImageGrid
+        images={[CollapsibleImg]}
+        caption={[
+            'Figure 1 displays the normal collapsed state of the menu.',
+            'Figure 2 displays the open state of the menu.',
+        ]}
+        regularWidth
+    />
+    <ImageGrid
+        images={[PersistentImg, PermanentImg]}
+        caption={
+            'When the user hovers on the Navigation Rail, the side navigation will expand temporarily, revealing the text details of each navigation item; when the user moves their cursor away, the whole side navigation panel collapses.'
+        }
+    />
+</React.Fragment>
+
+## Temporary
+
+A temporary drawer is invisible until it is opened — typically by pressing a button in the [App Bar](/patterns/appbar). This variation of the drawer appears on top of the main navigation, typically using a dark overlay to block interaction with the main content. Use this variation if you have:
+
+-   a mobile application
+-   extreme space limitations
+
+<React.Fragment>
+    <ImageGrid images={[NavigationTemporaryImg]} regularWidth />
+    <Divider />
+</React.Fragment>
+
+# Tab Navigation
+
+Tab navigation uses a horizontal bar with buttons for each navigation location. These can appear at the top or bottom of your application. On desktop and tablet, tabs at the top can be used in conjunction with drawer navigation as a way to navigate between sub-pages (e.g., you may have several sub-pages on your main settings page). Bottom tabs can be used in conjuction but is recommended that this pattern replaces the drawer navigation. On mobile it is also possible to combine top and bottom tabs on a single page.
+
+<ImageGrid images={[NavigationTabsImg]} regularWidth />
```

Comment:
Would be nice to have an image example of bottom navigation on mobile

Reply:
updated